tests/python/pants_test/engine/legacy/test_graph_integration.py

```diff
@@ -136,25 +117,30 @@ def setup_bundle_target(self) -> Iterator[None]:
       yield
 
   def test_missing_sources_warnings(self):
+    target_to_unmatched_globs = {
+      'missing-globs': ['*.a'],
+      'missing-rglobs': ['**/*.a'],
+      'missing-zglobs': ['**/*.a'],
+      'missing-literal-files': ['another_nonexistent_file.txt', 'nonexistent_test_file.txt'],
+    }
     with self.setup_sources_targets():
-      for target_name in self._SOURCES_ERR_MSGS.keys():
-        target_full = f'{self._SOURCES_TARGET_BASE}:{target_name}'
-        glob_str, expected_globs = self._SOURCES_ERR_MSGS[target_name]
```

Comment:
We weren't doing anything with the `glob_str` originally, so I removed it from `target_to_unmatched_globs` (previously `_SOURCES_ERR_MSGS `) for simplicity.